packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -283,9 +296,9 @@ The per protocol transaction timeout. Expired transactions will no longer be cor
 
 You can specify the following options in the `icmp` section of the +{beatname_lc}.yml+ config file:
 
-===== enabled
+===== enable
```

Comment:
No idea why this still shows as enabled. Locally it's `enabled`, only opening the github editor for file in feature branch shows `enabled`. Some github caching going on here?